test/liquid_voting_web/absinthe/mutations/create_delegation_test.exs

```diff
@@ -6,7 +6,9 @@ defmodule LiquidVotingWeb.Absinthe.Mutations.CreateDelegationTest do
 
   @new_delegator_email "new-delegator@email.com"
   @new_delegate_email "new-delegate@email.com"
+  @new_other_delegate_email "new-other-delegate@email.com"
```

Comment:
Might be worth renaming these variables now to improve legibility - I'd remove the `@new_` prepending all of them, so we're left with `@delegator_email`, `@delegate_email`, and `@another_delegate_email`